scripts/benchmark_wildcat.sh

```diff
@@ -99,7 +100,7 @@ for num_matched in ${num_matched_sizes[@]}; do
 					--output_root $OUTPUT_ROOT \
 					--max_resolution 760 \
 					2>&1 | tee $OUTPUT_ROOT/out.log
-				elif [[ $loader == *"olsson"* ]]
+				elif [[ $loader == *"colmap"* ]]
```

Comment:
What is this change for?

Reply:
There was a typo in the elif clause -- the string "olsson" was being used to check if we should run the colmap loader.